wiki_articles/special_members.md

```diff
@@ -1,24 +1,27 @@
 # Special Member Functions in C++
-?image https://cdn.discordapp.com/attachments/861906541823918101/877150143089098792/b2VBV.png
+<!-- image https://cdn.discordapp.com/attachments/861906541823918101/877150143089098792/b2VBV.png -->
```

Comment:
In the interest of making these files look close to the embed on github it might be good to not use a comment for this

Reply:
The reason why I decided against this for now is that there is extra implementation effort, and there isn't a `1:1` mapping that makes sense here. `![alt](url)` Markdown images come with an alt text, but this alt text would need to be discarded for embeds.

Furthermore, embeds also support videos, and I might add support for `<!-- video URL -->` if I find a good use case. This has the reverse problem: there is no counterpart in Markdown for embedding videos.

For now, I think it's okay to have this slight asymmetry.